bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -5702,6 +5685,7 @@ void ReplicaImp::sendResponses(PrePrepareMsg *ppMsg, IRequestsHandler::Execution
                                                                         req.outReplicaSpecificInfoSize,
                                                                         executionResult);
         if (replyMsg) {
+          LOG_INFO(GL, "Sending reply for req " << req.requestSequenceNum);
```

Comment:
move to debug (or remove). This will be sent for every client request

Reply:
done